### PR TITLE
profile programers first commit

### PR DESCRIPTION
**Pull request by cmcartaya**
I put more link in my profile 'couse you can realize about an error in the hight

### CODE REVIEW

File: cartaya/profile-programers/index.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    <title>profile-programers</title>
+</head>
+<body>
+    <div class="programer-profile">
+        <img class="programmer-profile_picture" src="./resources/images/sara.png" alt="profile picture">
+        <div class="programer-profile_info">
+            <p class="programer-profile_name lato-font color-font">saray pacheco</p>
+            <a class="programer-profile_mail lato-font" href="mailto:saraypacheco@gmail.com">saraypacheco@gmail.com</a>
+            <div class="programer-profile_challenge-container">
+                <div class="lalala">
```
Comment (TahimiLeonBravo) [inline cartaya/profile-programers/index.html]: What mean ```the class lalalala ```?. You need to follow the name convention

File: cartaya/profile-programers/resources/css/style.css [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+body {
+    align-items: center;
+    background-color: rgb(231, 234, 238);
+    display: flex;
+    height: 100vh;
+    justify-content: center;
+    margin: 0;
+}
+
+.lato-font {
```
Comment (TahimiLeonBravo) [inline cartaya/profile-programers/resources/css/style.css:18]: In this class, you only need use the basic styles of font. It should be maintainable easy.

File: cartaya/profile-programers/index.html (hunk 1 above)
Comment (cmcartaya) [reply cartaya/profile-programers/index.html]: ahhhhhhhh, sorryyy
I didnt get an idea for this class and I was putting that for a moment, and I forgot change it

File: cartaya/profile-programers/index.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./resources/css/style.css">
+    <title>profile-programers</title>
+</head>
+<body>
+    <div class="programer-profile">
+        <img class="programmer-profile_picture" src="./resources/images/sara.png" alt="profile picture">
+        <div class="programer-profile_info">
+            <p class="programer-profile_name lato-font color-font">saray pacheco</p>
```
Comment (TahimiLeonBravo) [inline cartaya/profile-programers/index.html]: You don't need create a class for the font color, put the style color in the class ```programer-profile_name```

**Review by TahimiLeonBravo — APPROVED** [review]

**Pull request merged**